test/presenters/return-requirements/site-description.presenter.test.js

```diff
@@ -0,0 +1,66 @@
+'use strict'
+
+// Test framework dependencies
+const Lab = require('@hapi/lab')
+const Code = require('@hapi/code')
+
+const { describe, it, beforeEach } = exports.lab = Lab.script()
+const { expect } = Code
+
+// Thing under test
+const SiteDescriptionPresenter = require('../../../app/presenters/return-requirements/site-description.presenter')
```

Comment:
As a [coding convention](https://github.com/DEFRA/water-abstraction-team/blob/main/coding_conventions.md#add-the-js-extension), we like to include the '.js' at the end of our files we require